crates/sovereign-sdk/module-system/sov-modules-stf-blueprint/src/lib.rs

```diff
@@ -625,7 +625,7 @@ where
         let mut previous_batch_hash = soft_confirmations[0][0].prev_hash();
         let mut last_commitment_end_height: Option<u64> = None;
 
-        let mut fork_manager = ForkManager::new(forks, sequencer_commitments_range.0 as u64);
+        let mut fork_manager = ForkManager::new(FORKS, sequencer_commitments_range.0 as u64);
```

Comment:
You did this now, but at some point i know i will undo your version because after discussing this with @eyusufatik  we want to be more flexible by passing the forks as some tests might want to use a different set of forks compared to the production ones. But for us to do this, we have to have a planned fork because passing `forks` into the verifier would require a fork (i know it's confusing)

Reply:
Only this one? I can fix it, not a big deal.

Reply:
No it's actually everywhere FORKS is used.